Automattic/jetpack
# Update CTA section on backup connection page

Adds an RNA prop to the connection layout to optionally update the design to RNA on a per use basis

Fixes #

## Proposed changes:

 * Add RNA prop to layout component
 * Update the backup connection cta module by passing the rna prop
 * New design includes color blobs and updated margins/padding

### Other information:


## Jetpack product discussion

P2: First part of this project: p1HpG7-kd7-p2
Figma: bSsQjyTPK8VIfopwI4fdIU-fi-1_29 

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch locally or with the Jetpack Beta plugin
2. DONT connect Jetpack to your wordpress account
3. Install the Jetpack VaultPress Backup plugin
4. If you are not redirected there, go to `/wp-admin/admin.php?page=jetpack-backup`
5. You should see the connection module with an updated design
Before:
![Pasted Graphic](https://github.com/Automattic/jetpack/assets/65001528/e20da69f-7ae4-4227-b95a-194afc137bbb)
After:
![image](https://github.com/Automattic/jetpack/assets/65001528/e6d1c15e-60ae-48d7-b195-3aca6b73c31f)
6. Make sure it scales to mobile and matches the figma comps on both mobile and desktop ( I played around with the position of the balls because the numbers in figma weren't producing quite the same result when I plugged them into the CSS )
